src/HonzaBotner.Discord.Services/EventHandlers/ReminderReactionsHandler.cs

```diff
@@ -0,0 +1,70 @@
+using System;
+using System.Threading.Tasks;
+using DSharpPlus.Entities;
+using DSharpPlus.EventArgs;
+using HonzaBotner.Discord.EventHandler;
+using HonzaBotner.Discord.Services.Options;
+using HonzaBotner.Services.Contract;
+using Microsoft.Extensions.Options;
+
+namespace HonzaBotner.Discord.Services.EventHandlers
+{
+    public class ReminderReactionsHandler : IEventHandler<MessageReactionAddEventArgs>
+    {
+        private readonly ReminderOptions _options;
+
+        private readonly IRemindersService _service;
+
+        public ReminderReactionsHandler(IOptions<ReminderOptions> options, IRemindersService service)
+        {
+            _options = options.Value;
+            _service = service;
+        }
+
+        public async Task<EventHandlerResult> Handle(MessageReactionAddEventArgs arguments)
+        {
+            // TODO: Move this to framework level code
```

Comment:
Not sure what do you mean by this, can you describe what changes you want to do?

Reply:
Imo events triggered by bot users shouldn't be handled, or at least I cannot find a reason for doing so

Reply:
My question was directed to moving of this code to framework level.

Reply:
Uh oh, my bad, I thought all events are dispatched by a single application entry point, so the check should be moved there